# Type: test
# Subtype: T1
# English Summary: Increase the timeout for DefaultStepRunnerTest#testParallelCommandFailure()
# Chinese Summary: 增加DefaultStepRunnerTest＃testParallelCommandFailure（）的超时时间
# Entry Number: 22

# === Git Diff Content ===

diff --git a/test/com/facebook/buck/step/DefaultStepRunnerTest.java b/test/com/facebook/buck/step/DefaultStepRunnerTest.java
index 391e0c1..1d40a1b 100644
--- a/test/com/facebook/buck/step/DefaultStepRunnerTest.java
+++ b/test/com/facebook/buck/step/DefaultStepRunnerTest.java
@@ -37,7 +37,7 @@ import java.util.concurrent.TimeUnit;
 
 public class DefaultStepRunnerTest {
 
-  @Test(expected=StepFailedException.class, timeout=1000)
+  @Test(expected=StepFailedException.class, timeout=5000)
   public void testParallelCommandFailure() throws Exception {
     ImmutableList.Builder<Step> commands = ImmutableList.builder();
     commands.add(new SleepingStep(0, 0));
